provider/core/src/varule_traits.rs

```diff
@@ -121,3 +121,14 @@ impl<T, const N: usize> MaybeEncodeAsVarULE for [T; N] {
         None
     }
 }
+
+#[cfg(feature = "export")]
+impl MaybeEncodeAsVarULE for u16 {
```

Comment:
question: why do we end up needing this?

Reply:
Issue: I think this impl is used in the efficient baked data struct encoder. This should definitely not return empty, or else the packed encoder won't work

Reply:
`u16` should not use a packed encoding. By default a type should use the normal encoding, certainly the primitives should.

Reply:
Yeah I agree that this should use the normal encoding.

Also, this trait is in `icu_provider`. I regret naming this `MaybeEncodeAsVarULE` instead of `MaybeEncodeForEfficientDataBake` or something. I confused myself :)

So this is not a problem.

Reply:
Right, we shouldn't have this impl; u16 shouldn't be handled differently here than most data structs.

Reply:
We need this impl if we want to use `u16` as a data struct, because all data structs need to implement this.